pubspec.yaml

```diff
@@ -4,13 +4,13 @@ version: 2.0.3
 homepage: https://github.com/zeshuaro/google_api_headers
 
 environment:
-  sdk: ">=2.12.0 <4.0.0"
+  sdk: '>=2.19.6 <4.0.0'
```

Comment:
Why do we need to bump the lower bound of the supported sdk and specifically to 2.19.6?